ui/src/shared/components/AddResourceDropdown.tsx

```diff
@@ -37,7 +37,7 @@ export default class AddResourceDropdown extends PureComponent<Props> {
     const {titleText, status} = this.props
     return (
       <Dropdown
-        style={{width: `${160}px`}}
+        style={{width: `${190}px`}}
```

Comment:
this can just be `'190px'`. no need to use string interpolation.

Reply:
Good catch